src/Socket/chats.ts

```diff
@@ -61,6 +61,69 @@ export const makeChatsSocket = (config: SocketConfig) => {
 		return privacySettings
 	}
 
+	/** helper function to run a privacy IQ query */
+	const privacyQuery = async(name: string, value: string) => {
+		await query({
```

Comment:
Is there a query response that you could parse so it's possible to see whether the query was successful?

Reply:
If the query is unsuccessful, the query function will show an error.
When it succeeds, the response does not receive enough data to return..and the response is not that important as this is used to give a command.. See other functions used by adiwajshing

Reply:
I compared it to the `fetchPrivacySettings` but you are right, `updateBlockStatus` for example worked the same way so that's fine